CONTRIBUTING.md

```diff
@@ -22,6 +22,18 @@ And if you have any questions, please feel free to reach out on [Discord].
 If you use editors that support [VS Code-style multi-root workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces),
 such as VS Code, Cursor...etc., opening the editor with `sentry-ruby.code-workspace` file will provide a better development experience.
 
+## Working in a devcontainer
+
+If you use [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension, you can open the project with the devcontainer by running `Remote-Containers: Reopen in Container` command.
+
+The devcontainer is configured with `.devcontainer/.env` file, that you need to create:
+
+```bash
+cp .devcontainer/.env.example .devcontainer/.env
```

Comment:
do we need to commit this `.env.example` file?

Reply:
@sl0thentr0py ohhh good catch TY :) just added it